Homework - OOP 3 Phone/src/main/java/Contact.java

```diff
@@ -0,0 +1,31 @@
+public class Contact extends Agenda{
+    private String contactName;
```

Comment:
there s no need for the Contact to extend Agenta:) A Contact is not an agenda:)
This class should be a simple class : public class Contact {....}